gobblin-core/src/main/java/org/apache/gobblin/writer/partitioner/TimeBasedWriterPartitioner.java

```diff
@@ -74,6 +76,7 @@
   private final String writerPartitionSuffix;
   private final DatePartitionType granularity;
   private final DateTimeZone timeZone;
+  @Getter
```

Comment:
required?

Reply:
Yes. We have a row check policy that requires time unit info from partitioner to compare record time with current time.